jenkins/bootstrap.py

```diff
@@ -684,6 +684,11 @@ def setup_magic_environment(job):
         'JENKINS_AWS_SSH_PUBLIC_KEY_FILE',
         os.path.join(home, '.ssh/kube_aws_rsa.pub'),
     )
+    os.environ.setdefault(
+        'K8S_SECURITY_SSH_PRIVATE_KEY_FILE',
```

Comment:
I added this field with the ssh key and the credential to jenkins but I am unsure if we want to inject this for all jobs, seems weird but I'm not sure where else to put it to make sure it only goes on that specific repos jobs